web/cobrands/northamptonshire/assets.js

```diff
@@ -253,9 +254,10 @@ fixmystreet.alloy_add_layers(northants_defaults, layers);
 
 // NCC roads layers which prevent report submission unless we have selected
 // an asset.
-var northants_road_defaults = $.extend(true, {}, fixmystreet.assets.alloyv2_defaults, {
+var northants_road_defaults = $.extend(true, {}, fixmystreet.alloyv2_defaults, {
     protocol_class: OpenLayers.Protocol.AlloyV2,
     http_options: {
+        base: "https://tilma.staging.mysociety.org/resource-proxy/proxy.php?https://northants.assets/${layerid}/${x}/${y}/${z}/cluster?styleIds=${styleid}",
```

Comment:
This should presumably use the is_live flag? Also presumably the "26" in the line below can't work, so does that want swapping with a placeholder or something? I see other commits have removed the need for is_live, is it still needed here? Ditto other places this changes.